marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/mastering/previewMatchingActivity.api

```diff
@@ -0,0 +1,144 @@
+{
+  "functionName": "previewMatchingActivity",
+  "params": [
+  {
+    "name": "sampleSize",
+    "datatype": "int"
+  },
+  {
+    "name": "uris",
+    "datatype": "string",
+    "multiple": true
+  },
+  {
+    "name": "stepName",
+    "datatype": "string"
+  }],
+  "return":
+  {
+    "datatype": "jsonDocument",
+    "$javaClass": "com.fasterxml.jackson.databind.JsonNode",
+    "description": "Returns the documentType with an 'id' matching the input",
+    "schema":
+    {
+      "$ref": "#/components/schemas/MatchActivityPreview"
+    }
+  },
+  "components":
```

Comment:
Does ML do anything with this? I wasn't aware that this is a thing

Reply:
Good question. I pasted this into the api file from the spec doc:
https://wiki.marklogic.com/display/ENGINEERING/Hub+Central+Mastering+Design+Specification